cmd/build.go

```diff
@@ -0,0 +1,125 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/boson-project/faas"
+	"github.com/boson-project/faas/buildpacks"
+	"github.com/boson-project/faas/docker"
+	"github.com/ory/viper"
+	"github.com/spf13/cobra"
+)
+
+var buildCmd = &cobra.Command{
+	Use:        "build",
+	Short:      "Build an existing function project as an OCI image",
+	SuggestFor: []string{"biuld", "buidl", "built", "image"},
+	// TODO: Add completions for build
+	// ValidArgsFunction: CompleteRuntimeList,
+	RunE: buildImage,
+	PreRunE: func(cmd *cobra.Command, args []string) (err error) {
+		flags := []string{"path", "tag", "push"}
+		for _, f := range flags {
+			err = viper.BindPFlag(f, cmd.Flags().Lookup(f))
+			if err != nil {
+				return err
+			}
```

Comment:
Ah; nice helper loop for that somewhat ugly BindPFlag call.
While this is a completely stylistic choice, these lines can also be written:
```
	if err = viper.BindPFlag(f, cmd.Flags().Lookup(f)); err != nil {
		return err
	}
```
... which saves a line at the expense of some slightly less readable ternary-ish syntactic sugar.